src/main/java/views/main_menu/MainMenuP.java

```diff
@@ -0,0 +1,22 @@
+package views.main_menu;
+
+import use_cases.main_menu.SessionSaveOB;
+import use_cases.main_menu.SessionSaveOD;
+
+/**
+ * The presenter class for the main menu.
+ * It is called by the interactor through the SessionSaveOB interface, and it is the
+ * class responsible for returning the SessionSaveOD objects
+ */
+public class MainMenuP implements SessionSaveOB {
+    /**
+     * Returns a SessionSaveOD signifying success which contains a success message to display
```

Comment:
Maybe rewrite this description to be a little clearer and less repetitive?

Reply:
Rewrote in new commit